CRM/Contribute/Form/Contribution.php

```diff
@@ -651,92 +651,32 @@ public function buildQuickForm() {
 
     $this->add('select', 'from_email_address', ts('Receipt From'), $this->_fromEmails);
 
-    $status = CRM_Contribute_PseudoConstant::contributionStatus();
-
-    // suppressing contribution statuses that are NOT relevant to pledges (CRM-5169)
-    $statusName = CRM_Contribute_PseudoConstant::contributionStatus(NULL, 'name');
```

Comment:
Removed variable `$statusName` is still in use at the [end of this function](https://github.com/civicrm/civicrm-core/blob/master/CRM/Contribute/Form/Contribution.php#L898) which would be undefined after the patch.